### ISSUE TITLE
mj-include: Specifying the path without .mjml results in warning when compiling

### ISSUE THREAD

**Issue by iainvdw** [mjmlio/mjml#336]
The documentation states that you can specify the path to the template to include without the .mjml extension, like `<mj-include path="./header" />`. However, when I do that, I get this warning: `Warning: Unregistered element: mj-include, skipping it`.

Adding the .mjml extenstion to the path does work.

Documentation states:

> <mj-include path="./header" />

**Comment by MatasSim** [comment]
pretty same here.. 

when I use `<mj-include path="order_confirm.mjml" />`  I get `Warning: Unregistered element: mjml, skipping it`.
when I use `<mj-include path="order_confirm" />` I get `Warning: Unregistered element: mj-include, skipping it`

MJML version - `3.8.6`
MJML utils version - `3.8.4`
gulp version - `3.9.1`
node version - `v4.4.4`

**Comment by iRyusa** [comment]
@MatasSim can you provide both mjml file for your first example ?

**Comment by iRyusa** [comment]
It looks like a regexp issue, I will try to fix it in 2.4 thanks for the feedback @iainvdw

**Comment by MatasSim** [comment]
sure,
first.mjml
```mjml
<mj-section>
  <mj-column>
    <mj-text>some text here</mj-text>
  </mj-column>
</mj-section>
```
second.mjml
```mjml
<mjml>
  <mj-body>
    <mj-container>
      <mj-include path="test" />
    </mj-container>
  </mj-body>
</mjml>
```

**Comment by selfagency** [comment]
I'm having this same issue with a brand new install. I've tried `header`, `header.mjml`, `./header`, `./header.mjml`, and even the full posix path. And each time, I get `Warning: Unregistered element: mj-include, skipping it`.

**Comment by romaniukjeff** [comment]
I'm also having the same issue with new install as of yesterday.

**Comment by iRyusa** [comment]
Can you both provide your mjml please ?

**Comment by selfagency** [comment]
my code was the same as @MatasSim's

**Comment by romaniukjeff** [comment]
Eh, I'm trying to actively move forward on this project but it was identical to your documentation example.

https://mjml.io/documentation/#mj-include

**Comment by iRyusa** [comment]
Can you try to set the absolute path of your included part ?

**Comment by selfagency** [comment]
As I specified above, I did.

**Comment by iRyusa** [comment]
Sorry haven't seen the full posix part, my bad.
I wonder why the regexp doesn't match mj-include at all..

**Comment by JordyPouw** [comment]
@iRyusa @selfagency @romaniukjeff 
Not sure if this is already fixed.. I had the same problem and the problem for me was the source location of the mjml files. 

Inside my project directory I created a 'src' and 'dest' directory. Whenever I ran the the mjml command or ran the gulp task I got same errors described as above. So I changed my include to: `<mj-include path="src/digest_email.mjml"/>`. 

The include probably reads from your root project directory and is not relative to the file that includes the partial file.
